src/extension.ts

```diff
@@ -176,8 +177,7 @@ export function startClient(
     client.onRequest(FSReadFile.type, (fsPath: string) => {
       return workspace.fs.readFile(Uri.file(fsPath)).then((uint8array) => new TextDecoder().decode(uint8array));
     });
-
-    runtime.telemetry.send({ name: 'yaml.server.initialized' });
+    runtime.telemetry.sendStartupEvent();
```

Comment:
can we send a boolean, for (un)successful startup?, and an error message in case of unsuccessful startup

Reply:
the `sendStartupEvent` doesn't have any parameters, we already send all LS errors, including startup errors.
Or you mean, do not use `sendStartupEvent` and instead send some   `{ name: 'yaml.server.initialized', successful : true/false }` ?

Reply:
the `sendStartupEvent` doesn't have any parameters, we already send all LS errors, including startup errors.
Or you mean, do not use `sendStartupEvent` and instead send some   `{ name: 'yaml.server.initialized', successful : true/false }` ?

Reply:
the `sendStartupEvent` doesn't have any parameters, we already send all LS errors, including startup errors.
Or you mean, do not use `sendStartupEvent` and instead send some   `{ name: 'yaml.server.initialized', successful : true/false }` ?

Reply:
@fbricon WDYT?

Reply:
You can manually craft a startup event like:
```json
{ 
name: 'startup', 
 properties: {
   'yaml.server.initialized' : true/false,
   'error': the error message if the server failed to initialize
 }
}
```